roles/dg-config/tasks/main.yml

```diff
@@ -16,14 +16,14 @@
 - name: DG | SQL to add standby logs
   set_fact:
     sql_for_dg:  |
-      alter system set dg_broker_start = true;
-      alter system set standby_file_management = auto;
+      alter system set dg_broker_start = true sid='*';
+      alter system set standby_file_management = auto sid='*';
```

Comment:
My understanding is that an "alter system" statement like this will update the spfile for all instances, but update in memory only for the instance currently connected to.  If this might run on a RAC system, do we need to either run the SQL on each node, or just do a "srvctl stop database && srvctl start database" type command?  (Though I'd hate to do the latter on a primary that may be running an active workload)

Reply:
by default it changes parameter in memory and spfile and those DG related parameters do not need reboot of instance to take effect. with sid='*' it would take care of all instances (even if it is single instance)

Reply:
Thanks;  just confirmed that at least in 19.6 this is the case.